src/Raven.Client/Documents/Commands/GetDocumentsCommand.cs

```diff
@@ -95,6 +109,19 @@ public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, Serve
                 }
             }
 
+            if (_includeAllCounters)
+            {
+                pathBuilder.Append($"&counter={Constants.Counters.All}");
+            }
+
```

Comment:
something went wrong with formatting here I think?

Reply:
what do you mean?